packages/qdrant-loader/src/qdrant_loader/core/pipeline/workers/upsert_worker.py

```
@@ -315,7 +315,9 @@ async def process_embedded_chunks(
             PipelineResult with processing statistics
         """
         logger.debug("UpsertWorker started")
-        logger.info(f"🔄 Starting upsert processing (max_workers={self.max_workers})...")
+        logger.info(
+            f"🔄 Starting upsert processing (max_workers={self.max_workers})..."
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)